journalapp/forms.py

```diff
@@ -0,0 +1,20 @@
+"""Define WTForm classes for adding and editing Entries into database."""
+
+from wtforms.validators import Length, InputRequired
+from wtforms import Form, StringField, TextAreaField
+
+
+class EntryForm(Form):
+    """Form used for both adding and editing Entry models."""
+
+    title = StringField(
+        'Title',
+        [Length(max=255, message='Your title is too long.'),
+         InputRequired(message='Title is required.')
```

Comment:
Why allow length 255 for your title form field when the database model will only allow 128 chars?